db/migrate/20180703023011_create_course_assessment_skills_question_assessments.rb

```diff
@@ -0,0 +1,29 @@
+class CreateCourseAssessmentSkillsQuestionAssessments < ActiveRecord::Migration[5.1]
+  def up
```

Comment:
Would it be better if we tried to rename and shift the columns around instead? Something like: 
 1. Add `question_assessment_id` foreign key, set to `null: true` first.
 1. Populate `question_assessment_id` with some SQL or ActiveRecord. 
 1. Set `question_assessment_id` to `null: false`
 1. Add proper indexes and remove old indexes, drop `question_id` FK column
 1. Rename table at the end

Reply:
As discussed, there would be no significant benefit for this approach over the currently implemented one of creating a new table.